src/opm/input/eclipse/EclipseState/InitConfig/InitConfig.cpp

```diff
@@ -120,6 +127,17 @@ namespace Opm {
 
         return this->equil;
     }
+    
+    bool InitConfig::hasStressEquil() const {
+        return !this->equil.empty();
+    }
+    
+    const DeckKeyword& InitConfig::getStressEquil() const {
+        if( !this->hasStressEquil() )
+            throw std::runtime_error( "Error: No 'EQUIL' present" );
```

Comment:
Error message is wrong (actually, due to the bug above it is correct, unintentionally...)

Reply:
Fixed.